src/main/java/org/elasticsearch/search/aggregations/bucket/histogram/InternalHistogram.java

```diff
@@ -310,6 +310,10 @@ public Type type() {
         return factory;
     }
 
+    public Rounding getRounding() {
+        return emptyBucketInfo.rounding;
+    }
```

Comment:
yep it is currently. exposing this method will help prediction aggregations (such as moving average) so they can use the rounding to generate the keys for the buckets they wish to add. Sorry, this is a bit of an aside to the main purpose of the PR but something we said we would also need.